schema.json

```diff
@@ -230,6 +230,77 @@
       },
       "required": ["fileName"]
     },
+    "normalPayload": {
+      "definitions": {
+        "temperature": {
+          "type": "number",
+          "description": "Temperature (°C)",
+          "min": -273.15
+        },
+        "direction": {
+          "type": "number",
+          "description": "Direction (°)",
+          "min": 0,
+          "max": 360
+        },
+        "speed": {
+          "type": "number",
+          "description": "Speed (m/s)",
+          "min": 0
+        },
+        "measurement": {
+          "type": "object",
+          "properties": {
+            "time": {
+              "type": "string",
+              "format": "date-time",
+              "description": "Date and time of the measurement (RFC3339)"
+            },
+            "air": {
+              "type": "object",
+              "properties": {
+                "temperature": {
+                  "description": "Air temperature (°C)",
+                  "$ref": "#/definitions/normalPayload/definitions/temperature"
+                },
+                "relativeHumidity": {
+                  "type": "number",
+                  "description": "Relative humidity (%)",
+                  "min": 0,
+                  "max": 100
+                },
+                "pressure": {
+                  "type": "number",
+                  "description": "Atmospheric pressure (hPa)"
+                }
```

Comment:
Even though sane values are [900..1100] otherwise we're seeing world records, I think we should limit min/max values to unit validity and allow room for wrong measurements.

The wind cannot come from 470°, the relative humidity cannot be 120%, the wind speed cannot be negative. Likewise, a temperature of -500℃ doesn't exist either.

But.... What happens when something is off with the temperature sensor though? Is it an acceptable consequence to not consider the result normal payload, and not process it as such? Or would we leave an outlier of -280℃ to the application layer to handle?

Reply:
I gave this another thought; I think it's up to the normalizer function to validate stuff. Naively converting a voltage in a temperature without bounds checking is just wrong. We have `errors` and `warnings` output for this as well.